app/src/main/java/com/vuongnq14798/vuongmp3/ui/home/HomeFragment.kt

```diff
@@ -87,10 +60,10 @@ class HomeFragment : BaseFragment(),
     }
 
     override fun onGenreClicked(genre: Genre) =
-        startActivity(context?.let { DetailGenreActivity.getIntent(it, genre) })
+        startActivity(DetailGenreActivity.getIntent(requireContext(), genre))
```

Comment:
dùng requireContext() thì chẳng khác gì context!! cả, 
```suggestion
       context?.let{ startActivity(DetailGenreActivity.getIntent(it, genre)) }
```